src/askscript/__tests__/04-control-flow/program11-for_check_limit.test.ts

```diff
@@ -0,0 +1,9 @@
+import program from './program11-for_check_limit.ask';
+
+test('computes', async () => {
+  const maxArgUnderLimit = 587;
```

Comment:
Aaaah, so there is no obvious relationship between AskVM limit and number of iterations of an empty loop.